addons/crm/tests/test_crm_lead.py

```diff
@@ -758,3 +758,58 @@ def test_phone_mobile_update(self):
         self.assertEqual(lead.phone, self.test_phone_data[1])
         self.assertEqual(lead.mobile, self.test_phone_data[2])
         self.assertFalse(lead.phone_sanitized)
+
+    @users('user_sales_manager')
+    def test_crm_lead_alias_help(self):
+        crm_team_model = self.env['ir.model']._get_id('crm.team')
+        crm_lead_model = self.env['ir.model']._get_id('crm.lead')
+
+        # archive other teams
+        self.env['crm.team'].search([]).active = False
+
+        user_team_leads, team_leads, user_team_opport, team_opport = self.env['crm.team'].create([{
+            'name': 'UserTeamLeads',
+            'use_leads': True,
+            'member_ids': [(6, 0, [self.env.user.id])],
+            'company_id': self.env.company.id,
```

Comment:
Seems like this is already the default, so if not relevant I wouldn't specify it. 

Also maybe we can have a test for different companies too (i.e. not suggesting an alias from a different company), since that should be fixed too. Maybe just adding a step to this test.